### PR TITLE
US922204: Remove marathon & docker swarm from Autoscaler

### PR DESCRIPTION
**Pull request by caoimhelynch**
https://internal.almoctane.com/ui/entity-navigation?p=131002/6001&entityType=work_item&id=922204

### CODE REVIEW

**Comment by andyreidz** [comment]
@tonymcveigh Any idea why the documentation site link isn't being generated for this PR?

File: release-notes-2.2.0.md [hunk 1]
```diff
@@ -4,5 +4,7 @@
 ${version-number}
 
 #### New Features
+- **US922204**: Autoscaler is only available using Kubernetes
```
Comment (rorytorneymf) [inline release-notes-2.2.0.md]: Think this is a breaking change?

Comment (caoimhelynch) [reply release-notes-2.2.0.md]: Yeah I wasn't sure about this, will change to breaking change thanks.

**Comment by rorytorneymf** [comment]
There is a .project file I think can be removed as well

**Comment by rorytorneymf** [comment]
> @tonymcveigh Any idea why the documentation site link isn't being generated for this PR?

@caoimhelynch Can you follow this up? I see see references to Docker swarm in the below link, thanks.

https://github.houston.softwaregrp.net/sepg-deploy-qa/Autoscaler_CI_autoscaler_US922204/tree/deploy/quick-start

**Review by rorytorneymf — APPROVED** [review]

**Pull request merged**